src/dfx-core/src/config/model/dfinity.rs

```diff
@@ -1,32 +1,34 @@
 #![allow(dead_code)]
-use crate::config::dfinity::MetadataVisibility::Public;
-use crate::lib::bitcoin::adapter::config::BitcoinAdapterLogLevel;
-use crate::lib::canister_http::adapter::config::HttpAdapterLogLevel;
-use dfx_core::config::directories::get_config_dfx_dir_path;
-use dfx_core::error::dfx_config::DfxConfigError;
-use dfx_core::error::dfx_config::DfxConfigError::{
+#![allow(clippy::should_implement_trait)] // for from_str.  why now?
```

Comment:
Other than module reorganization, this was the only source change required in this refactor.  I don't understand why it's needed now.